app/models/manageiq/providers/workflows/automation_manager/configuration_script_source.rb

```diff
@@ -1,30 +1,42 @@
 class ManageIQ::Providers::Workflows::AutomationManager::ConfigurationScriptSource < ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationScriptSource
-  FRIENDLY_NAME = "Embedded Workflows Repository".freeze
+  FRIENDLY_NAME     = "Embedded Workflows Repository".freeze
+  BUILTIN_REPO_NAME = "ManageIQ Built-in Workflows".freeze
+
+  supports :update do
+    _("Cannot update the built-in repository") if name == BUILTIN_REPO_NAME
+  end
+  supports :delete do
+    _("Cannot delete the built-in repository") if name == BUILTIN_REPO_NAME
+  end
 
   def self.display_name(number = 1)
     n_('Repository (Embedded Workflows)', 'Repositories (Embedded Workflows)', number)
   end
 
+  def self.seed
+    return unless Settings.prototype.ems_workflows.seed_builtin_workflows
+
+    manager = ManageIQ::Providers::Workflows::AutomationManager.in_my_region.first
+    return if manager.nil?
+
+    manager.configuration_script_sources
```

Comment:
Why the .tap?  I assume you used it to return the seeded record but I thought we didn't care about the return value?

Reply:
Yeah that's true, it was returning the seeded repository previously but we don't need to

Reply:
Done